Fitter/scripts/EFTPlotter.py

```diff
@@ -185,6 +191,28 @@ def GetWCsNLLFromRoot(self,base_name_lst,wc,unique=False):
         return [graphwcs,graphnlls]
 
 
+    def CreateNewLimitTreefor2DScan(self,name_lst,wc_lst):
+        graphwcs, graphnlls = self.GetWCsNLLFromRoot(name_lst,wc_lst,unique=True)
+        newRootFile = ROOT.TFile("tmp.root","RECREATE")          #temporary root file with new limit tree
```

Comment:
That's a good idea. I think my logic back then was that this function would allow us to dump out a root file (if need be), and we could investigate the TTree inside this file if we wanted to. Since this was 2 years ago, I definitely didn't have as good of an understanding of the code as I have now, so I would probably have opted for a different way.